docs/1.2/transaction_family_specifications/identity_transaction_family.md

```diff
@@ -34,23 +34,19 @@ policy is updated, all roles referencing the policy are updated as well.
 An example role might be named \"transactor\", and references the policy
 that controls who is allowed to submit batches and transactions on the
 network. This example is further described in
-`Transactor Permissioning <../sysadmin_guide/configuring_permissions>`{.interpreted-text
-role="doc"}.
+[Transactor
+Permissioning]({% link docs/1.2/sysadmin_guide/configuring_permissions.md%}).
```

Comment:
Does this need a space before `%}`?

Reply:
I added the space, but it renders correctly without it